lib/gemstash/S3.rb

```diff
@@ -0,0 +1,247 @@
+require "gemstash"
+require "digest"
+require "fileutils"
+require "yaml"
+require "aws-sdk-s3"
+require "pathname"
+
+module Gemstash
+  class S3
+    extend Gemstash::Env::Helper
+    attr_reader :folder, :client, :bucket
+
+    VERSION = 1
```

Comment:
Maybe call this `STORAGE_VERSION`?

Reply:
Addressed in 8c8713bf5f9b37e8a2e8e889a3f614e25141b9de